auth/README.md

```diff
@@ -0,0 +1,13 @@
+# How to deploy
+1. Clone bahis-oidc in the same level as bahis-infra
+```bash
+git clone https://github.com/road86/bahis-oidc
```

Comment:
This needs updating now we are using submodules

Reply:
Ohh. Forgot to edit readme. doing